packages/forms-web-app/src/dynamic-forms/options-question.js

```diff
@@ -93,18 +93,20 @@ class OptionsQuestion extends Question {
 				optionData.checked = (',' + answer + ',').includes(',' + optionData.value + ',');
 			}
 
-			// also need to handle dependent fields & set their answers
-			// if (optionData.conditional !== undefined) {
-			// 	let conditionalField = { ...optionData.conditional };
-			// 	conditionalField.fieldName = this.fieldName + '_' + conditionalField.fieldName;
-			// 	conditionalField.value = journey.response.answers[conditionalField.fieldName] || '';
-			// 	optionData.conditional = {
-			// 		html: nunjucks.render(
-			// 			`../views/questions/conditional/${conditionalField.type}.njk`,
-			// 			conditionalField
-			// 		)
-			// 	};
-			// }
+			// handle conditional (dependant) fields & set their answers
+			if (optionData.conditional !== undefined) {
+				let conditionalField = { ...optionData.conditional };
+
+				conditionalField.fieldName = this.fieldName + '_' + option.value + '-value';
```

Comment:
Optional but if you want you could write it this way:
``${conditionalField.fieldName}_${option.value}-value`;`

Reply:
I've gone with the approach above to make it slightly clearer (imo) how the fieldname is constructed